Lib/fontTools/ttLib/__init__.py

```diff
@@ -879,8 +879,8 @@ def _escapechar(c):
 		return hex(byteord(c))[2:]
 
 
-def nameToIdentifier(name):
```

Comment:
NOTE: these changes revert the tagToIdentifier to previous implementation b/c we no longer need the nameToIdentifier function exposed.  After this commit, there should be no changes to this function since prior to the split_glyphs branch.